rclc_parameter/QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,220 @@
+
+The stability of `rclc_parameter` is ensured through reviews, CI and tests.
+The change control process can be found in [CONTRIBUTING](../CONTRIBUTING.md).
+
+All changes to `rclc_parameter` occur through pull requests that are required to pass all CI tests.
+In case of failure, only maintainers can merge the pull request, and only when there is enough evidence that the failure is unrelated to the change.
+Additionally, all pull requests must have at least one positive review from another contributor that did not author the pull request.
+
+### Change Requests [2.i]
+
+All changes will occur through a pull request.
+
+### Contributor Origin [2.ii]
+
+This package uses [Developer Certificate of Origin (DCO)](https://developercertificate.org/) as its confirmation of contributor origin policy since version 1.0.0. More information can be found in [CONTRIBUTING](../CONTRIBUTING.md).
+
+### Peer Review Policy [2.iii]
+
+All pull requests will be peer-reviewed by at least one other contributor who did not author the pull request. Approval is required before merging.
+
+### Continuous Integration [2.iv]
+
+All pull requests must pass CI to be considered for merging, unless maintainers consider that there is enough evidence that the failure is unrelated to the changes.
+CI testing is automatically triggered by incoming pull requests.
+Current results can be seen [../README.md](../README.md).
+
+###  Documentation Policy [2.v]
+
+All pull requests must resolve related documentation changes before merging.
+
+## Documentation [3]
+
+### Feature Documentation [3.i]
+
+`rclc_parameter` features are documented in the package in the header files.
+
+### Public API Documentation [3.ii]
+
+`rclc_parameter` has embedded API documentation.
+
+### License [3.iii]
+
+The license for `rclc_parameter` is Apache 2.0, and a summary can be found in each source file.
+A full copy of the license can be found [here](../LICENSE).
+
+### Copyright Statements [3.iv]
+
+The copyright holders each provide a statement of copyright in each source code file in `rclc_parameter`.
+
+## Testing [4]
+
+### Feature Testing [4.i]
+
+`rclc_parameter` provides tests which simulate typical usage, and they are located in the [`test` directory](test).
+New features are required to have tests before being added as stated in [CONTRIBUTING](CONTRIBUTING.md).
+Current results can be seen [../README.md](../README.md).
+
+Most features in `rclc_parameter` have corresponding tests which simulate typical usage, and they are located in the [`test`](./test) directory.
+New features are required to have tests before being added.
+
+### Public API Testing [4.ii]
+
+Each part of the public API has tests, and new additions or changes to the public API require tests before being added. The tests aim to cover both typical usage and corner cases, but are quantified by contributing to code coverage.
+
+### Coverage [4.iii]
+
+Coverage checks for `rclc_parameter` have not been implemented.
```

Comment:
@pablogs9 codecov is enabled for all packages of rclc: That includes also rclc_parameters. 
e.g. https://codecov.io/gh/ros2/rclc/tree/galactic/ros_ws/src/0oshwe2aqpon/rclc/rclc_parameter/src/rclc_parameter

Reply:
Thanks